From 343854360cb2af97316fb11c6e0d288aca17cdf2 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Fri, 11 Dec 2020 11:46:40 +0800
Subject: [PATCH] ARM: rockchip_defconfig: enable CONFIG_VM_EVENT_COUNTERS by
 default.

Test:
    CtsAppExitTestCases
    android.app.cts.ActivityManagerAppExitInfoTest#testLmkdKill

Android R Go's lmkd needs this configuration to correctly match vmstat,
enable it to make lmkd work properly.

Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
Change-Id: I57ee648f70f7b44755f7ddb0d8bb3082273e926b
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index dea6528df85d..5b33bf107588 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -37,7 +37,6 @@ CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 # CONFIG_FHANDLE is not set
 CONFIG_BPF_SYSCALL=y
 CONFIG_EMBEDDED=y
-# CONFIG_VM_EVENT_COUNTERS is not set
 # CONFIG_SLUB_DEBUG is not set
 # CONFIG_COMPAT_BRK is not set
 CONFIG_PROFILING=y
-- 
2.35.3

